driver/pelican/types/pelican.go

```diff
@@ -28,24 +28,47 @@ var stateMappings = map[string]int32{
 // TODO Support case where the thermostat is configured to use Celsius
 
 type Pelican struct {
-	username string
-	password string
-	name     string
-	target   string
-	timezone *time.Location
-	req      *gorequest.SuperAgent
+	username      string
+	password      string
+	Name          string
+	HeatingStages int32
+	CoolingStages int32
+	TimezoneName  string
```

Comment:
This is included so we can recover timezone info when reading Pelican data from the database upon driver startup.